### DISCUSSION TITLE
Is there a way to fullscreen over 2 displays?

### DISCUSSION THREAD

**Discussion by andris1** [raysan5/raylib#2936] [Q&A]
As a hacky workaround I've been able to extend a borderless window across 2 monitors, but was wondering if there's a better way. I want to do essentially split screen, but using 2 monitors.

**Comment by mallory-scotton** [comment]
True fullscreen across two monitors isn't really possible in the traditional sense, OS fullscreen mode always locks to a single display. But the borderless window approach you're already using is actually the **correct solution**, not a hack.

This is the approach used by games like BeamNG.drive for example.
